_data/shafi1996/shafi.md

```diff
@@ -0,0 +1,7 @@
+---
+name: MUHAMMED SHAFI PM
+institution: T JOHN INSTITUTE OF TECHNOLOGY
+profile_pic: Shafi1996.png
```

Comment:
Space in the file name, please do one of the following:
1. Change `Shafi1996.png` to `Shafi1996 .png` in the markdown file.
2. Rename `Shafi1996 .png` to `Shafi1996.png`.
I'd suggest you rename the file!

Reply:
Thanks for mentioning it. I will do that.

Reply:
Hey,  could you Please check now?